setup.cfg

```diff
@@ -111,9 +112,11 @@ markers =
     remote_data: marks this test function as needing remote data.
     online: marks this test function as needing online connectivity.
     mpl_image_compare: marks this test function as using hash-based Matplotlib figure verification. This mark is not meant to be directly applied, but is instead automatically applied when a test function uses the @sunpy.tests.helpers.figure_test decorator.
+    remote_status: mark this test function as reporting back status of remote requests.
```

Comment:
So how does one use this? We have to mark all our remote-data tests with it?

Reply:
This marker is for the test which pings the urls; so that we can call it easily.

For intercepting the urls, once ``--intercept-remote`` option is issued, urllib, request and socket are monkeypatched for the entire session, so we won't have to mark each test.